### DISCUSSION TITLE
Non Time series panel

### DISCUSSION THREAD

**Discussion by gianfrancodemarco** [HadesArchitect/GrafanaCassandraDatasource#170] [Q&A] [answered]
Hello, 

I have a very basic question: this plugin allows only to build time series panels or can be used with other type of panels?
I'm trying but I am not succeeding to use different panel types.

Thanks

**Comment by HadesArchitect** [comment, marked as answer]
Ciao Gianfranco!

This issue is addressed in #167 which is hopefully lands this or next week 😁

Hai una buona settimana!

**Reply by gianfrancodemarco** [reply]
That's awesome!

I'm pretty newbie with Grafana and Plugins, but please let me know if I can help in any way.

Just out of curiosity, this implementation will allow for the use of the table panel alone or other type of panels are planned?

Thanks, have a good day

**Reply by unflag** [reply]
@gianfrancodemarco It is more like table-format data support, than support of a specific panel. I believe, that any panel using standard grafana features should work properly with the datasource. Feel free to raise an issue if you encounter any problems with any panel.
